### ISSUE TITLE
[BUG] Need to adjust LD_LIBRARY_PATH after upgrading pandas

### ISSUE THREAD

**Issue by tebeka** [rapidsai/cudf#692]
**Describe the bug**

Using rapids docker. After `conda install pandas=0.23.4` `import cudf` fails since it can't find `libNVStrings.so`.

**Steps/Code to reproduce bug**
```
$ docker run --runtime=nvidia --rm -it rapidsai/rapidsai:cuda9.2_ubuntu16.04
$ source activate rapids
$ conda install pandas=0.23.4
$ ipython
In [1] import cudf
In [2] df = cudf.DataFrame()
In [3] df['x'] = [1,2,3]
```

**Expected behavior**
Working with cudf should work

**Environment details (please complete the following information):**
 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
rapids docker image `rapidsai/rapidsai:cuda9.2_ubuntu16.04`

 - Method of cuDF install: [conda, Docker, or from source]
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used
 - Please run and attach the output of the `cudf/print_env.sh` script to gather relevant environment details
 

**Additional context**
Adding `/conda/pkgs/nvstrings-0.0.3-cuda9.2_py35_18/lib` to `LD_LIBRARY_PATH` fixes the problem.

**Comment by randerzander** [comment]
Should be fixed after https://github.com/rapidsai/cudf/pull/668 merges

**Comment by kkraus14** [comment]
Fixed by #668.
